core/account-id/tests/borsh/main.rs

```diff
@@ -0,0 +1,14 @@
+use bolero::check;
+use borsh::{BorshDeserialize, BorshSerialize};
+use near_account_id::AccountId;
+
+fn main() {
+    check!().for_each(|input: &[u8]| {
```

Comment:
Should we merge this two as well? I'd rathre merge

Reply:
We could, but the cargo-bolero custom test harness is quite a bit more appealing to the eye: while the libtest harness shows only one line for the 1000 subtests performed, the cargo-bolero harness shows one dot per actually-performed test. You can try it out by running `cargo test -p near-account-id borsh` vs. `cargo test -p near-vm-runner wasmer2_and_wasmtime_agree`, the first one shows progress while the second one just waits for 10 seconds (here) before saying "all good".

In this specific case the fuzzer is very fast so it's probably less big a deal anyway, but I kind of wanted to use this one as a showcase of "how people should write fuzzers by default, with a minimal example." Do you have strong opinions against rust integration tests or is it just "with all else equal unit tests are better"?

Reply:
>Do you have strong opinions against rust integration tests

In this case, not, because the crate is very small. But in general yeah, I have strong opinions against integrated tests, because they increase linking time a lot for larger crates. 

>is quite a bit more appealing to the eye

I think I'd prioritise UX of "here's a file with standard `#[test]` functions" over eye-candy in the terminal, though no strong opinion here

Reply:
Makes sense! Then I've just pushed the account-id tests to unit tests too, and will still use them as the default example for how to write a fuzzing test :)